.travis.yml

```diff
@@ -148,6 +150,16 @@ matrix:
           env: MAIN_CMD='pycodestyle astroquery --count'
                SETUP_CMD='' EVENT_TYPE='push pull_request'
 
+        - stage: deploy
+          deploy:
+              provider: pypi
+              user: "keflavich"
```

Comment:
do we want maybe to set up an auto deploy user that will only have access to astroquery rather your whole pypi?

Reply:
Could do that for sure.  I guess we can just set up a user called 'astroquery'?